### ISSUE TITLE
Suggestion: remember scoll

### ISSUE THREAD

**Issue by LeonardoGentile** [HappenApps/Quiver#29]
Sometimes I produce very long notes, so if I am at the bottom of it and I want to temporary switch to another note, when I go back I'll have to scroll again all the way down (or anywhere I was at before)

It would nice to have an option to remember the position where we were before closing/switching to another note.

**Comment by ylian** [comment]
Good idea but sometimes you do want to see the note scroll to top. Adding a setting is possible, but seems excessive. Have to think about it.

**Comment by neerajbadlani** [comment]
+1 , i context switch a lot between notes , will be huge time saver if Quiver has this feature
